paddle/fluid/primitive/composite/composite.h

```diff
@@ -206,6 +206,12 @@ Tensor pow_decomp(const Tensor& x, const paddle::Scalar& y) {
   }
 }
 
+template <typename T>
+Tensor reciprocal_decomp(const Tensor& x) {
+  Tensor index = full<T>(empty_shape, -1.0, x.dtype());
+  return elementwise_pow<T>(x, index);
```

Comment:
for better performance, use divide instead.

Reply:
done